stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -432,6 +434,62 @@ public Source createSourceSynchronous(
                 null, mContext, params, apiKey, mStripeAccount, mLoggingResponseListener);
     }
 
+    /**
+     * Blocking method to retrieve a {@link PaymentIntent} object.
+     * Do not call this on the UI thread or your app will crash.
+     *
+     * @param paymentIntentParams a set of params with which to retrieve the Payment Intent
+     * @param publishableKey a publishable API key to use
+     * @return a {@link PaymentIntent} or {@code null} if a problem occurred
```

Comment:
I recognize this is an existing documentation pattern, but I'm immediately curious what kinds of problems would occur that don't result in an exception being thrown.

Reply:
Looks like it's only if there's a problem deserializing the `PaymentIntent` from the response, or the api key is blank?

Reply:
I think you can get like a 402?